hybris/glesv2/glesv2.c

```diff
@@ -678,6 +679,13 @@ static void glGetShaderSource_wrapper(GLuint shader, GLsizei bufsize, GLsizei* l
 
 static const GLubyte* glGetString_wrapper(GLenum name)
```

Comment:
Any reason why we can't merge this _wrapper into the actual function at the bottom?

Otherwise LGTM